### PR TITLE
Added new version of LOS rings, 3 draws is now 2

### PR DESCRIPTION
**Pull request by Borg-King**
Have renamed original file "_old", as well as changing widget name and enabled=False for testing FPS on other systems.
gui_sensor_ranges_los_old.lua can be removed subsequently

Testing changes to GUI_sensor_ranges_los:
4000 Ticks on screen (strategic view, only icons and LOS rings):
Average FPS no LOS rings: 232
Average FPS original LOS rings: 156
Average FPS new LOS rings: 168

### CODE REVIEW

File: luaui/Widgets/gui_sensor_ranges_los.lua [hunk 1]
```diff
@@ -11,11 +11,12 @@ function widget:GetInfo()
 end
 
 -------   Configurables: -------------------
-local rangeColor = { 0.9, 0.9, 0.9, 0.24 } -- default range color
+local rangeColor = { 0.9, 0.9, 0.9, 0.74 } -- default range color
```
Comment (Beherith) [inline luaui/Widgets/gui_sensor_ranges_los.lua:14]: Why was this changed, is the appearence different?

**Comment by Beherith** [comment]
Also, is there a reason to keep the old one, given git history?

File: luaui/Widgets/gui_sensor_ranges_los.lua (hunk 1 above)
Comment (Borg-King) [reply luaui/Widgets/gui_sensor_ranges_los.lua:14]: Oh yes that change wasn't necessary and can be reverted!

**Comment by Borg-King** [comment]
It was so that one could have both available in BAR to test FPS differences but not needed. Commit has removed

**Comment by Beherith** [comment]
Thank you very much!
